app/routes/profile/$email/index.tsx

```diff
@@ -114,67 +171,204 @@ export const ProfileInfo = () => {
+                    </Box>
+                  )}
                 </Box>
+              </Box>
+            </Paper>
+          </Grid>
+          <Grid item xs={12} md={8}>
+            {profileData.projectMembers?.length > 0 && (
+              <Paper elevation={0} sx={{ padding: 2 }}>
+                <h2 style={{ marginTop: 0, paddingLeft: 20 }}>Projects</h2>
+                <Grid container sx={{ p: 2 }}>
+                  {profileData.projectMembers.map((projectMember) => (
+                    <Grid item xs={12} key={projectMember.id}>
+                      <Card
+                        key={projectMember.id}
+                        sx={{ marginBottom: 3, display: "block" }}
+                      >
+                        <CardContent>
+                          <Typography gutterBottom variant="h5" component="div">
+                            {projectMember.project.name}
+                          </Typography>
+                          <Typography variant="body2" color="text.secondary">
+                            {projectMember.project.description}
+                          </Typography>
+                          <Typography variant="body2" color="text.secondary">
+                            {projectMember.role
+                              .map((role: { name: string }) => role.name)
+                              .join("/")}
+                            {" - "}
+                            {projectMember.hoursPerWeek} hours per week
+                          </Typography>
+                          {projectMember.practicedSkills?.length > 0 && (
+                            <Box
+                              sx={{
+                                display: "flex",
+                                flexDirection: "row",
+                                gap: "8px",
+                                marginTop: "1rem",
+                              }}
+                            >
+                              {projectMember.practicedSkills.map((skill) => (
+                                <Chip label={skill.name} key={skill.id}></Chip>
+                              ))}
+                            </Box>
+                          )}
+                        </CardContent>
+                      </Card>
+                    </Grid>
+                  ))}
+                </Grid>
               </Paper>
-            </Grid>
-            <Grid item xs={12} md={9}>
+            )}
+            {githubProjects && githubProjects.length > 0 && (
               <Paper elevation={0} sx={{ padding: 2 }}>
                 <h2 style={{ marginTop: 0, paddingLeft: 20 }}>
-                  Active Projects
+                  Github Active Projects
                 </h2>
                 <Grid container sx={{ p: 2 }}>
-                  {githubProjectsLink.map((project) => (
+                  {githubProjects.map((project) => (
                     <Grid item xs={12} key={project.id}>
                       <Card
```

Comment:
also could be nice that the card projects will be clickable

Reply:
Fixed!